db/migrate/20230822184619_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo, default: "https://this-person-does-not-exist.com/img/avatar-gen11828689f76739ac5772010de9c529f7.jpg"
+      t.text :Bio , default: "Developer"
+      t.datetime :CreatedAt
+      t.datetime :UpdatedAt
+      t.integer :PostCounter, default: 0
+
+      t.timestamps
```

Comment:
- [ ] [Optional] [Highly recommended] Please remove columns `:CreatedAt` and `:UpdatedAt` because `t.timestamps` at the end will create it for you automatically.

- [ ] You should use snake_case for naming database columns according to the [Rails naming conventions](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e#database). Please name this column `:posts_counter`, instead of `:PostCounter`.